py/strato/common/log/log2text.py

```diff
@@ -396,11 +396,28 @@ def executeRemotely(args):
     return runRemotely(args.node, ignoreArgs)
 
 
+def minimumLevel(minLevel, noDebug):
+    if minLevel is None:
+        return logging.INFO if noDebug else logging.DEBUG
+    level_dict = {'debug': logging.DEBUG,
+                  'info': logging.INFO,
+                  'warning': logging.WARNING,
+                  'error': logging.ERROR,
+                  'progress': logging.PROGRESS,
+                  'success': logging.SUCCESS,
+                  'step': logging.STEP}
+    for string, level in level_dict.iteritems():
+        if string.startswith(minLevel.lower()):
+            return level
+    return logging.DEBUG
+
+
 if __name__ == "__main__":
     import argparse
     parser = argparse.ArgumentParser()
     parser.add_argument("logFiles", metavar='logFile', nargs='*', help='logfiles to read')
     parser.add_argument('-d', "--noDebug", action='store_true', help='filter out debug messages')
+    parser.add_argument('-l', '--min-level', action='store', default='', help='minimal log level to display (substring is OK, case-insensitive)')
```

Comment:
curios: why not defining `-v / --verbose` with `action=count` (see: https://docs.python.org/2/library/argparse.html#action)? 
this way every occurrence of `-v` will increase the logging level to display.